scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -935,7 +1066,7 @@
         [/filter_second]
 #ifdef DEBUG_MODE
         # (same string as before; no additional comment)
-        {MSG_narrator _"Using debug mode to kill Bor Cryne is cheating!"}
+        {MSG_narrator _"Using debug mode to kill Bor Cryne is cheating! Any narrative inconsistencies that arise from having done so are entirely your fault."}
```

Comment:
I think being cheeky and calling the player a cheater is good enough ahahaha.

Reply:
The added sentence was meant to address feedback from Marximilian, who told me that he hoped it didn't break lore of the story when he did that

Reply:
It's fine you don't need to add it because you have another message that tells you Bor Cryne isn't supposed to die. Not sure how Marx missed that.

Reply:
ok removed in [d9dba4b](https://github.com/nemaara/A_New_Order/pull/43/commits/d9dba4bfd0c5918bc3d493d42d8d5d32782419ea)